### DISCUSSION TITLE
Not loaded

### DISCUSSION THREAD

**Discussion by razclud** [astrandb/miele#576] [Q&A] [answered]
I am a beginner in HA and i just installed my miele integration but it does not recognize any devices and I don't know where to begin to fix this issue :(

![2024-11-03 04_09_49-Settings – Home Assistant — Mozilla Firefox](https://github.com/user-attachments/assets/97317bb6-6b92-4670-80a1-01d320ade85b)

**Comment by astrandb** [comment]
Please take a look in the HA log and see if there are any entries from miele or pymiele.
You can also try to upgrade you integration to 2024.9.0b1 - Open the HACS panel and select "Redownload" on the Miele entry, select version v2024.9.0b1 and restart HA. This new release adds some more detailed logging during startup.

**Reply by razclud** [reply]
Thank you for the reply. I did update it, same issue. This is what its said in the logbook 

![2024-11-04 02_13_37-Logbook – Home Assistant — Mozilla Firefox](https://github.com/user-attachments/assets/ffc2ff49-f64d-4543-92ce-e87daec233c5)

I deleted the integration but it did not disappear, a configure button appeared and i logged in again and associated HA with miele. It said the it cannot in red letters and that i should restart HA. 

![2024-11-04 02_20_38-Developer tools – Home Assistant — Mozilla Firefox](https://github.com/user-attachments/assets/05c77e88-7ff1-4df3-a72e-94e66df9d6ef)

This is my yaml


# Loads default set of integrations. Do not remove.
default_config:

# Load frontend themes from the themes folder
frontend:
  themes: !include_dir_merge_named themes

automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml
miele:
    client_id: <833e20b4-52db-4fab-b756-bebbca27778a>
    client_secret: <F61epgu7GMsp2cMYBhmqtszVnmvhg7mn>
    lang: <optional. en=english, de=german>
    cache_path: <optional. where to store the cached access token>


![2024-11-04 02_28_20-Settings – Home Assistant — Mozilla Firefox](https://github.com/user-attachments/assets/52f48292-42a9-4ddb-acd6-09959d893415)

**Comment by astrandb** [comment, marked as answer]
I think you have used installation instructions from wrong repo. This one is listed in HACS as Miele - A modern integration for Miele devices in Home Assistant. You should not enter anything - no miele:... in configuration.yaml. So please try to clear out miele stuff in configuration.yaml and restart HA and then readd the Miele integration.

**Comment by razclud** [comment]
THANK YOU

It worked, all good now
